frontend/helpers/mock-to-officer-type.ts

```diff
@@ -0,0 +1,83 @@
+import { OfficerRecordType, EmploymentType } from "../models/officer"
+import { Incident, Officer, UseOfForce } from "../helpers/api"
+import officers from "../models/mock-data/officer.json"
+
+export function getOfficerFromMockData(officerId: number) {
+  if (officerId >= 0 && officerId < 100) {
+    return mockToOfficerType(officers[officerId])
+  }
+}
+
+export function mockToOfficerType(officer: typeof officers[0]): OfficerRecordType {
+  function mockToWorkHistoryType(workHistory: typeof officer.workHistory): EmploymentType[] {
+    let converted: EmploymentType[] = []
```

Comment:
lets make this into a more declarative `map` function like the one in a previous comment